### DISCUSSION TITLE
change window border color

### DISCUSSION THREAD

**Discussion by Jeffmagma** [tauri-apps/tauri#9469] [Q&A]
After creating a new tauri v2 app, I made the window undecorated, and noticed that even though the title bar is gone, the border is still there, and it matches the accent color in windows:
![image](https://github.com/tauri-apps/tauri/assets/7827718/2e421ac2-e002-4034-a1c3-124419915d3e)
How can I change the color or remove this border?

**Comment by FabianLars** [comment]
Afaik this requires you to disable the shadows too ("shadows": false).

**Reply by Jeffmagma** [reply]
so there's no way to keep the native shadows + rounded edges but remove the border like other apps (that i think are built in electron?) has it? is it something that's possible to be added to tauri?

**Reply by FabianLars** [reply]
In the underlying windows apis it is possible. It's just not correctly implemented in Tauri/Tao.

**Reply by Jeffmagma** [reply]
Could this be added as a feature request 😅

**Reply by FabianLars** [reply]
Sure, could you open an issue for this then? :)

**Reply by Type-32** [reply]
any progress on this? is the issue created

**Reply by Jeffmagma** [reply]
I actually just disabled the border thing in the windows settings instead of trying to do it programmatically since I was basically the only one using my application, but you're free to create the issue if you haven't already!

**Reply by keytrap-x86** [reply]
> I actually just disabled the border thing in the windows settings instead of trying to do it programmatically since I was basically the only one using my application, but you're free to create the issue if you haven't already!

How did you disable the border ?

**Reply by Jeffmagma** [reply]
windows settings > personalization > colors > scroll down past the big color palette

<img width="870" height="98" alt="image" src="https://github.com/user-attachments/assets/3767041b-1d86-43bb-b731-8bb89a6d7cc9" />
